vite.config.js

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from 'vite';
+import laravel from 'laravel-vite-plugin';
+
+export default defineConfig({
+    plugins: [
+        laravel({
+            input: [
+                'resources/css/tailwind.css',
+                // 'resources/js/site.js',
```

Comment:
Currently commented out, because no site.js exists.
In a fresh Statamic application, there always exists a site.js, so that could also be activated if wanted.